zkevm-circuits/src/super_circuit/test.rs

```diff
@@ -20,7 +20,7 @@ fn super_circuit_degree() {
     SuperCircuit::configure_with_params(&mut cs, params);
     log::info!("super circuit degree: {}", cs.degree());
     log::info!("super circuit minimum_rows: {}", cs.minimum_rows());
-    assert!(cs.degree() <= 9);
+    assert!(cs.degree() <= 12);
```

Comment:
Note: In this PR, the super_circuit degree is 12.

Reply:
A quick fix to reduce degree from 12 -> 10 in this pr https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1508. After pr is appoved and merged will update this pr as well

Reply:
Updated in latest commit. 

Final degree `9 -> 10`